src/main/kotlin/no/nav/syfo/melding/kafka/PublishUbesvartMeldingService.kt

```diff
@@ -19,7 +22,10 @@ class PublishUbesvartMeldingService(
     fun publishUbesvartMelding(
         meldingTilBehandler: MeldingTilBehandler,
     ) {
-        // TODO: Send på kafka
+        kafkaUbesvartMeldingProducer.sendUbesvartMelding(
+            kafkaUbesvartMeldingDTO = meldingTilBehandler.toKafkaUbesvartMeldingDTO(),
```

Comment:
Hadde det vært bedre om man bare sendte inn `meldingTilBehandler` her slik at produceren har ansvar for å lage key og konvertere til kafka-objekt?